### DISCUSSION TITLE
upgrade from nextjs 11 to 13 for metadata api won't work

### DISCUSSION THREAD

**Discussion by albjeremias** [vercel/next.js#51942] [Help] [answered]
### Summary

I need help with making the new metadata api to work with my code

### Additional information

```js
I can't get the metadata to work on my project.. i just upgraded to the latest nextjs.

you can check reproduce it using this branch of my repository:
https://github.com/helpbuttons/helpbuttons/compare/seo

I added this code to my page:


export const metadata: Metadata = {
  title: 'Home',
  description: 'Welcome to Next.js',
}
```

and the title is not set. im sure i did something badly with the upgrade, but i really need support cause i have no idea where to start debugging this. thanks
```


### Example

https://github.com/helpbuttons/helpbuttons/compare/seo

**Comment by icyJoseph** [comment, marked as answer]
Hi,

The MetaData API, as far as I know, only works in the App Directory. You are using it in the Pages Directory, and there it won't have any effect.

In Pages Dir: https://nextjs.org/docs/pages/building-your-application/optimizing#metadata this applies
